lib/util.ml

```diff
@@ -1753,7 +1753,36 @@ let hexa_string s =
   done;
   Bytes.unsafe_to_string s'
 
-let print_alphab_list conf crit print_elem liste =
+let sort_by_date base pl =
+  List.sort
+    (fun x1 x2 ->
+      match Gutil.alphabetic (p_surname base x1) (p_surname base x2) with
+      | 0 -> (
+          match
+            ( Date.od_of_cdate (get_birth x1),
+              Date.od_of_cdate (get_birth x2) )
+          with
+          | Some d1, Some d2 -> Date.compare_date d1 d2
+          | Some _, _ -> 1
+          | _ -> -1)
+      | n -> -n)
+    pl
+
+let print_alphab_list conf base crit print_elem liste =
+  let print_sub_list base len sub_l print_e index =
```

Comment:
build error for the same reason: base not used because line 1774 is a comment line.